README.md

```diff
@@ -41,6 +41,8 @@ import 'package:flutter_circular_chart/flutter_circular_chart.dart';
 Create chart data entry objects:
 
 ```
+final GlobalKey<AnimatedCircularChartState> _chartKey;
```

Comment:
Place this in its own code block with a leading comment explaining why we need it, something like:

    Create a [GlobalKey](https://docs.flutter.io/flutter/widgets/GlobalKey-class.html) to be able to access the chart and update its data:
    
    ```
    final GlobalKey<AnimatedCircularChartState> _chartKey;
    ```
    
    Create chart data entry objects:
    ...